v23.1/cockroach-auth-session.md

```diff
@@ -95,7 +95,8 @@ To run any of the `auth-session` subcommands, you must be a member of the [`admi
 
 - The `login` subcommand allows users with the [`admin` role](security-reference/authorization.html#admin-role) to create HTTP authentication tokens with an arbitrary duration. If operational policy requires stricter control of authentication sessions, you can:
 
-  - Monitor the `system.web_sessions` table for all current and recent HTTP sessions.
+  - Monitor the `system.web_sessions` table for all current and recent HTTP sessions. If you monitor this table regularly, consider adjusting the `server.log_gc.period` and
```

Comment:
Adjusting it how? I don't find this suggestion actionable. I guess if you have to ask you shouldn't be doing it? Or is this worth unpacking a bit

Reply:
Ah, in this case, I don't have any additional info for these two cluster settings beyond their [spartan descriptors already public](https://www.cockroachlabs.com/docs/stable/cluster-settings.html), which I link to. I don't have any actionable advice at all in fact; just here to raise awareness.